starter-code/functions-and-arrays.js

```diff
@@ -23,27 +36,57 @@ var words = [
 var longest = findLongestWord(words);
 console.log(longest);
 
+//==============================================================
 // Calculating a Sum
 function sumArray (array) {
-
+  var theSum=0;
+  for (var i = 0; i < numbers.length; i++) {
+    theSum += numbers[i];
+    console.log(theSum);
+  }
 }
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 var total = sumArray(numbers);
 console.log(total);
 
-// Calculate the Average
-function averageNumbers (array) {
+// ALTERNATIVE METHOD CALCULATING A SUM
+
+function getSum(total, num) {
+    return total + num;
+}
+function sumArray(item) {
+    return numbers.reduce(getSum);
+}
```

Comment:
Here you have the method we talked about it yesterday where you combine both functions:

function sumArray (array) {
  return array.reduce(function (sum, number){
    return sum + number;
  }, 0);
}